sampleExtension/statistics/exerciseTime.js

```diff
@@ -0,0 +1,60 @@
+
+function ExerciseTime() {
+    var self = this;
+
+    this.init = function() {
+        // Make sure there is a record for the current day by calling this method.
+        self.incrementTodayExerciseTime(0);
+    };
+
+
+    // Increments the counter for time spent on exercises today with 'amount'.
+    // When there is no currentDateStatistic in the storage yet, create one.
+    // And when the currentDateStatistic does not correspond with the actual current date,
+    // this currentDataStatistic is added to the list of previous day statistics.
+    this.incrementTodayExerciseTime = function(amount){
+        storage.getTodayExerciseTime().then(function(response){
+            self.handleIncExerciseTime(response.tds_exerciseTimeToday, amount);
```

Comment:
It used to be the case that every storage function got a callback that would be called by the storage as soon as the function got the element from the storage.  This way you'd prevent having to use things like output.tds_blacklist or in this case response.tds_exerciseTimeToday. This is still the case for the settings and blacklist I saw, but why is it not the case here? it seems in-consequent with the rest. So do you have an explanation or possible improvement?
Why not make response (which comes from the storage I assume) response.tds_exerciseTimeToday by default?